patterns/ecs-v1/haproxy

```diff
@@ -30,10 +25,15 @@ HAPROXYCAPTUREDRESPONSEHEADERS %{DATA:captured_response_headers}
 # HAPROXYCAPTUREDREQUESTHEADERS %{DATA:request_header_host}\|%{DATA:request_header_x_forwarded_for}\|%{DATA:request_header_accept_language}\|%{DATA:request_header_referer}\|%{DATA:request_header_user_agent}
```

Comment:
nit: even though these are commented out, we should update the examples to use ECS-ish names.